build.cake

```diff
@@ -27,7 +27,7 @@ var packageVersion = version + modifier + dbgSuffix;
 //////////////////////////////////////////////////////////////////////
 
 var WindowsFrameworks = new string[] {
-    "net-4.5", "net-4.0", "net-3.5", "net-2.0", "portable" };
+    "net-4.5", "net-4.0", "net-3.5", "net-2.0", "netstandard", "portable" };
 
```

Comment:
The `PackageZip` task will also needs updating here. I think it can probably refer to this frameworks list, rather than have a duplicate!